modules/meeting/app/forms/meeting/time_group.rb

```diff
@@ -41,7 +41,8 @@ class Meeting::TimeGroup < ApplicationForm
         required: true,
         autofocus: false,
         data: {
-          action: "input->recurring-meetings--form#updateFrequencyText"
+          action: "input->recurring-meetings--form#updateFrequencyText \
+                   input->meetings--form#updateTimezoneText"
```

Comment:
Almost. The string you created is `"input->recurring-meetings--form#updateFrequencyText                    input->meetings--form#updateTimezoneText"`. It would be better to have `"input->recurring-meetings--form#updateFrequencyText input->meetings--form#updateTimezoneText"` without all the spaces

Ruby allows to concatenate string without any `+` between the strings:
```
irb
>> "hello"   "world"
=> "helloworld"
```

And `\` escape the following character, meaning that if it's used at the end of a line, it will ignore the newline character and consider that the 2 lines are on the same line.

So the above can be written like this:
```suggestion
          action: "input->recurring-meetings--form#updateFrequencyText " \
                   "input->meetings--form#updateTimezoneText"
```
This way it's only one string. This syntax is a bit strange but valid and recognized by rubocop like in [Layout/LineContinuationSpacing cop](https://docs.rubocop.org/rubocop/cops_layout.html#layoutlinecontinuationspacing) for instance.

Reply:
Aah, I missed the trailing space

I initially added it in the way you suggested, with two strings in quotes, but that broke the functionality. I assumed I got the syntax wrong and so made it one string, but now with the trailing space it works fine too, so I guess I missed that initially. Thanks for the info anyway! I hope I can remember to do it like this in the future instead of all in one line